pkg/chunk/cassandra/storage_client.go

```diff
@@ -48,6 +51,9 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 	f.StringVar(&cfg.Password, "cassandra.password", "", "Password to use when connecting to cassandra.")
 	f.DurationVar(&cfg.Timeout, "cassandra.timeout", 600*time.Millisecond, "Timeout when connecting to cassandra.")
 	f.DurationVar(&cfg.ConnectTimeout, "cassandra.connect-timeout", 600*time.Millisecond, "Initial connection timeout, used during initial dial to server.")
+	f.IntVar(&cfg.Retries, "cassandra.retry-count", 0, "Number of retries to perform on a request. (Default is 0: no retries)")
```

Comment:
`retry-count ` > `max-retries` may be more clear and consistent across the config

Reply:
good callout will adjust it.